docs/whatsnew/2202.1.0.rst

```diff
@@ -29,6 +29,7 @@ Object changes
 * Added System Object (:pull:`73`)
 * Add to Installer: InstallerName (:pull:`74`)
 * Add Forecast, ForecastScope objects (:pull:`79`)
+* Add Scope, Measurement, EnergyACMeas, IrradGlobalHorizMeas, PowerACMeas, TempAmbMeas (:pull:`82`)
```

Comment:
We need to move this to a new file 2202.1.1.rst and not modify 2202.1.0.rst.

I'd say "Renamed ForecastScope to Scope"

Have we removed ForecastScopeType and its ItemType?